platform/util/platform-util/pom.xml

```diff
@@ -90,6 +90,10 @@
             <artifactId>common-system</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shiro</groupId>
+            <artifactId>shiro-core</artifactId>
```

Comment:
✏️ We need to be careful adding dependencies to platform-util since the transitive dependencies can impact anyone using this utility library. 

❓ Is there a security util that already uses shiro that we could use instead?

Reply:
Looks like I could move the ThreadContextUtils to security-core-services to avoid the platform-utils change.